charts/opentelemetry-operator/values.schema.json

```diff
@@ -858,6 +859,53 @@
                 "runAsUser": 65534,
                 "fsGroup": 65534
             }]
+        },
+        "testFramework": {
+            "type": "object",
+            "default": {},
+            "title": "The testFramework Schema",
+            "required": [
+                "image"
```

Comment:
"tag" is also required I believe based on the usage of the settings.  If we don't have a tag we'd end up with `<image-namge>:`

Reply:
Ya you're right I was in the wrong spot